### ISSUE TITLE
Can't get v0.4.1 working

### ISSUE THREAD

**Issue by brotankfrank** [JTNDev/galaxy-integration-wii#10]
None of my games show up. Reverted back to v0.3.4 and it works normally.

**Comment by JTNDev** [comment]
Thanks! I'll try to work it out. Could you please try changing the best
match game setting in the user config off and see if that works? It'll help
me try to troubleshoot.

**Comment by JTNDev** [comment]
I've figured out the problem and I'll upload a new version in a few minutes
from now

**Comment by brotankfrank** [comment]
Nice i will reply back when I'm home from work and let you know how it goes. 

Thanks for the swift response!

**Comment by brotankfrank** [comment]
I dont see a download link in the v0.4.2 sections. Only source code

**Comment by JTNDev** [comment]
It seems like GitHub didn't upload the file correctly. I reuploaded it now
but in this case it doesn't really matter whether you download the source
code or the zip.

**Comment by develoqqer** [comment]
does this mean issue #3 is fixed?

**Comment by JTNDev** [comment]
Yes it does. I closed it on the GC version of this plugin but I forgot to
close it on the Wii
